modules/sc-mesh-secure-deployment/configure.sh

```diff
@@ -58,44 +82,59 @@ cat <<EOF > tools/wpa_tools/access_point.conf
     frequency=2437
 }
 EOF
-cd tools/wpa_tools
-chmod +x create_access_point.sh
-sudo bash create_access_point.sh $choice $ip
-cd ../..
+# FIXME: Each time you echo this it's gonna add to the end of the files and
+#        create duplicate lines
+echo "INTERFACES=$1" >> /etc/default/isc-dhcp-server
```

Comment:
It may contain multiple interfaces so probably a bad idea to overwrite. I like the backup and recovery though, this could be a way forward. Will create an issue.